mmv1/third_party/terraform/services/compute/go/resource_compute_network_firewall_policy_rule_test.go

```diff
@@ -250,6 +430,8 @@ resource "google_compute_network_firewall_policy_rule" "fw_policy_rule1" {
   security_profile_group = "//networksecurity.googleapis.com/${google_network_security_security_profile_group.security_profile_group_updated.id}"
   direction              = "INGRESS"
   disabled               = false
+  tls_inspect            = true
```

Comment:
Why did we add this field to the config?

Reply:
Lets keep this identical to prior to the change.

Reply:
If we remove both `tls_inspect` field the check will complain that is missing coverage